fast64_internal/f3d/f3d_material.py

```diff
@@ -2513,42 +2522,49 @@ class RDPSettings(bpy.types.PropertyGroup):
         items=enumAlphaDither,
         default="G_AD_NOISE",
         update=update_node_values_with_preset,
+        description="Applies your choice dithering type to output frambuffer alpha"
     )
     # v2 only
     g_mdsft_rgb_dither: bpy.props.EnumProperty(
         name="RGB Dither",
         items=enumRGBDither,
         default="G_CD_MAGICSQ",
         update=update_node_values_with_preset,
+        description="Applies your choice dithering type to output frambuffer color"
     )
     g_mdsft_combkey: bpy.props.EnumProperty(
         name="Chroma Key",
         items=enumCombKey,
         default="G_CK_NONE",
         update=update_node_values_with_preset,
+        description="Turns on/off the chroma key. Chroma key requires a special setup to work properly"
     )
     g_mdsft_textconv: bpy.props.EnumProperty(
         name="Texture Convert",
         items=enumTextConv,
         default="G_TC_FILT",
         update=update_node_values_with_preset,
+        description="Turns on/off the converter for texture color. Can convert color space from YUV into RGB"
```

Comment:
This is not very useful and bordering on incorrect. I'd suggest "Sets the function of the texture convert unit, to do texture filtering, YUV to RGB conversion, or both"